src/components/Checkout/SocialShare.vue

```diff
@@ -88,20 +97,29 @@ export default {
 	},
 	data() {
 		return {
-			selectedLoanIndex: 0,
+			copyStatus: {
+				class: '',
+				disabled: false,
+				text: 'Copy Link'
+			},
+			maxMessageLength: 280,
 			message: '',
-			maxMessageLength: 280
+			selectedLoanIndex: 0,
 		};
 	},
 	computed: {
+		selectedLoan() {
+			return this.loans[this.selectedLoanIndex];
```

Comment:
We don't always have a loan on the thanks page, so it would be good to include a fallback here like `return this.loans[this.selectedLoanIndex] || {};`

Reply:
👍 I merged [a PR](https://github.com/kiva/ui/pull/1413) this morning that conditionally displays this social block only if we have loans.

Reply:
That's good! We should still write this code defensively in case this component gets used somewhere that forgets to add that check. "Cannot read x of undefined" errors are difficult to track down, so it's good practice to write components that can handle their props being malformed. Lodash.get can help a lot with that, especially when you need to get a deeply-nested piece of data.